packages/account-v2/src/components/Timeline/Timeline.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ComponentProps, HTMLAttributes, PropsWithChildren, ReactElement } from 'react';
+import { twMerge } from 'tailwind-merge';
+import { Text } from '@deriv-com/ui';
+
+type TTimelineItemProps = HTMLAttributes<HTMLDivElement> & { itemTitle: React.ReactNode };
+
+type TTimelineProps = HTMLAttributes<HTMLOListElement> & {
+    children: ReactElement<TTimelineItemProps>[];
+    lineHeight?: ComponentProps<typeof Text>['lineHeight'];
+};
+
+const TimelineItem = ({ children }: PropsWithChildren<TTimelineItemProps>) => {
+    return <div>{children}</div>;
+};
+
+const Marker = ({ label }: { label: number }) => {
+    return (
+        <div className='border absolute w-24 h-24 pl-px border-solid-red-0 rounded-full bg-solid-red-0 leading-[23.5px] mr-8 text-center -left-12'>
```

Comment:
Can u check `text-center -left-12` I think `-left-12` is not valid

Reply:
Yeah @likhith-deriv, I realised this. I was refactoring it